salt/metalk8s/internal/sreport/init.sls

```diff
@@ -0,0 +1,2 @@
+include:
+  - .installed
```

Comment:
I think you need to include it somewhere in the deployment of the nodes to install it anywhere automatically. Maybe part of the `minion` roles

Reply:
as it is in internal, make sens to include it in its init.sls ?

Reply:
We never run the entire `metalk8s.internal`. 
And btw this sosreport package it's not "internal" you should move it directly to `salt/metalk8s`. 
`internal` directory is more for salt internal stuff. That mean states that should not be call directly.